src/app/(proper_react)/redesign/(authenticated)/user/dashboard/page.tsx

```diff
@@ -9,10 +9,7 @@ import { View } from "./View";
 import { authOptions } from "../../../../../api/utils/auth";
 import { dashboardSummary } from "../../../../../functions/server/dashboard";
 import { getCountryCode } from "../../../../../functions/server/getCountryCode";
-import {
-  getSubscriberBreaches,
-  guidedExperienceBreaches,
-} from "../../../../../functions/server/getUserBreaches";
+import { getSubscriberBreaches } from "../../../../../functions/server/getUserBreaches";
```

Comment:
It's fine to remove this now.. Though we will need to fetch it somewhere for the guided experience data

Reply:
Yes, but right now, this was really crowding my logs, and after removing it complaining about an unused import. We can reimport it when we implement the guided experience :)